### DISCUSSION TITLE
Product Flow Map

### DISCUSSION THREAD

**Discussion by f20180445** [EXERLOG/exer_log#121] [General]
Hi @KalleHallden , it's great to see so many folks help you out with the startup since you've gone open source. From a code perspective, I believe there aren't many difficulties understanding parts of it (and I'm hoping it'll be much easier once the software architecture is in place with riverpod).

However, I was hoping if you could create (or share if you already have), a document that explains the entirety of the app. I'm a product guy, and in our language we call it a PRD (Product Requirements Doc), that helps the developers understand what is the purpose of the app, what features it includes, the user flow on the app (how a user would move from one screen to another) and a lot more. (For reference, check - [this](https://www.prodroll.com/post/product-requirements-document-prd-with-example))

Once you set the base, others can catch on to your idea a lot quicker and help you develop and suggest features, hence making the UI/UX discussions more meaningful. (Suggestion - You can make a video about it if you feel that makes it easier for to explain the app and the features and link it here)

I would love to help you set up product development processes once I completely understand your idea for the app.

**Comment by KalleHallden** [comment]
Yeah this is a great idea. I am in a bit of an overwhelming situation with so many comment threads to deal with for these discussions, issues, PR's etc so it's quite difficult to figure out where to place my energy first. Right now I've focused maybe too much on PR's. I think what you're suggesting might be the best place to start. Shocking as it may sound, I have almost no experience working on something like this, so I wouldn't really be sure where to start. 

But I will actually re-prioritize for a couple days and try to create this document. Then you might be able to help me review, refine and make that more clear.

**Reply by f20180445** [reply]
Sure thing, I'll be waiting for you to share the doc here. And we're all learning as we build this :) I understand that you're overwhelmed by all the comments out here, what I can suggest is that you identify individuals who can dedicate some time here and appoint them as the "manager" for a particular thing - for eg. Discussions or PR or Issues, etc.

At the end of the day/alternate days, you can connect with them on call to get a quick review of what's happening in each of those verticals here.

This way, things can be structured, you'll not be overwhelmed with everything at once (but maybe overwhelmed one vertical at a time :P), and you will have time to focus on things that you're good at and prioritise that.

PS - it's more or less like running a company with individuals willing to contribute, and it's not a good idea for one person to be doing everything from my experience

**Reply by KalleHallden** [reply]
Yeah that would be a good idea. I've at least set up maintainers now, so we'll see what happens there :)

I created a document here for the PRD: https://docs.google.com/document/d/172ofqs4MEg5aDWP-tBBRm6QFhPE9R7KzAaDSs3C4wAA/edit?usp=sharing 
I think it is better than the way it's been before where we haven't had much structure but I am unsure of how to refine this document to something that is maybe even more useful
